CHANGES.md

```diff
@@ -127,6 +125,8 @@ This change also aligned the authentication architectures between DC/OS Enterpri
 
 * Fix IPv6 VIP support in L4LB (DCOS-50427)
 
+* DC/OS UI X-Frame-Options default value has been changed to `DENY` from `SAMEORIGIN` (DCOS-49594)
```

Comment:
I'd prefer 
```
from `SAMEORIGIN` to `DENY`
```

Also, should this indicate that it is now configurable, as before, for upgrades from 1.12.{0-3} to 1.13?

Reply:
I'd suggest:
```
DC/OS UI X-Frame-Options default value has been changed from `SAMEORIGIN` to `DENY`. This is now configurable using the `adminrouter_x_frame_options` configuration value.
```